src/extension/geofence/src/main/resources/applicationContext.xml

```diff
@@ -127,4 +127,12 @@
           class="org.geoserver.geoserver.authentication.GeoFenceXStreamPersisterInitializer"
           lazy-init="false">
     </bean>
+
+    <bean class="org.geoserver.platform.ModuleStatusImpl">
```

Comment:
Since you need to set capabilities it would be probably better \ cleaner to implement a concrete instance of ModuleStatus for GeoFence, e.g. GeoFenceModuleStatus?

Reply:
fixed